ui/repetition.reel/repetition.js

```diff
@@ -1942,94 +1942,32 @@ var Repetition = exports.Repetition = Component.specialize(/** @lends Repetition
             this._startX = 0;
             this._startY = 0;
 
-            var document = this.element.ownerDocument;
-
-            if (window.Touch) {
-                document.removeEventListener("touchend", this, false);
-                document.removeEventListener("touchmove", this, false);
-                document.removeEventListener("touchcancel", this, false);
+            this.__pressComposer.removeEventListener("press", this, false);
+            this.__pressComposer.removeEventListener("pressCancel", this, false);
 
-            } else {
-                document.removeEventListener("mouseup", this, false);
-                document.removeEventListener("mousemove", this, false);
-            }
+            this.element.removeEventListener("touchmove", this, false);
```

Comment:
Symmetric comment, why do we still need this here even though we've abstracted most of the work through a composer?